test/python/evaluation/test_output_results.py

```diff
@@ -0,0 +1,40 @@
+from test.python.evaluation import TARGET_XLSX_DATA_FOLDER, XLSX_DATA_FOLDER
+from test.python.evaluation.testing_config import get_parser
+from typing import Union
+
+import pandas as pd
+import pytest
+from src.python import MAIN_FOLDER
+from src.python.common.tool_arguments import RunToolArguments
+from src.python.evaluation.evaluation_config import EvaluationConfig
+from src.python.evaluation.xlsx_run_tool import create_dataframe
+
+FILE_NAMES = [
+    ('test_sorted_order.xlsx', 'target_sorted_order.xlsx', False),
+    ('test_sorted_order.xlsx', 'target_sorted_order.xlsx', True),
+    ('test_unsorted_order.xlsx', 'target_unsorted_order.xlsx', False),
+    ('test_unsorted_order.xlsx', 'target_unsorted_order.xlsx', True),
+]
+
+
+@pytest.mark.parametrize(('test_file', 'target_file', 'output_type'), FILE_NAMES)
+def test_correct_output(test_file: str, target_file: str, output_type: Union[bool, str]):
+
+    parser = get_parser(RunToolArguments)
+    parser.add_argument('-xlsx_file_path', default=XLSX_DATA_FOLDER / test_file)
+    parser.add_argument('-tp', '--tool-path', default=MAIN_FOLDER.parent / 'review/run_tool.py')
+    if output_type:
+        parser.add_argument('-tr', '--traceback', action='store_false')
+    else:
+        parser.add_argument('-tr', '--traceback', action='store_true')
+
+    args = parser.parse_args([])
+    config = EvaluationConfig(args)
```

Comment:
you can use just a dict with necessary values, you should not use argument parser here